app/models/user.rb

```diff
@@ -1,13 +1,23 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :rememberable, :validatable, :confirmable, :recoverable
+
```

Comment:
- [ ] email verification addedvia devise confirmable :+1:

Reply:
@VICTOR-CHUKWU  thank you